website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -61,7 +61,7 @@ the traffic redirection command is automatically set up via an init container.
 
 * To use transparent proxy on Kubernetes, Consul-helm >= `0.32.0` and Consul-k8s >= `0.26.0` are required in addition to Consul >= `1.10.0`.
```

Comment:
```suggestion
* Consul Helm chart 0.32.0 or later 
* Consul on Kubernetes 0.26.0 r later
* Consul 1.10.0 or later
```
There are other issues with how this is organized, but let's address some low-hanging fruit and use some conventional language while we're at it.